packages/perseus/src/widgets/interactive-graphs/axis-ticks.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from "react";
+
+import {useTransform} from "./graphs/use-transform";
+
+import type {vec} from "mafs";
+
+const tickSize = 10;
+
+const tickStyle: React.CSSProperties = {
+    stroke: "black",
+    strokeWidth: 1,
+};
+
+const YGridTick = ({y}: {y: number}) => {
+    const startingPoint: vec.Vector2 = [0, y];
+    let [startPtPx] = useTransform(startingPoint);
+
+    const endPtPx = [startPtPx[0] + tickSize / 2, startPtPx[1]];
+    startPtPx = [startPtPx[0] - tickSize / 2, startPtPx[1]];
+
+    return (
+        <g className="y-axis-ticks" tabIndex={0} style={{} as any}>
+            <line
+                x1={startPtPx[0]}
+                y1={startPtPx[1]}
+                x2={endPtPx[0]}
+                y2={endPtPx[1]}
+                style={tickStyle}
+            />
+        </g>
+    );
+};
+
+const XGridTick = ({x}: {x: number}) => {
+    const startingPoint: vec.Vector2 = [x, 0];
```

Comment:
This `startingPoint` is really the middle of the tick mark, right? I wonder if something like this, without the reassignment, would be clearer:

```ts
const pointOnAxis: vec.Vector2 = [x, 0];
const [[xPx, yPx]] = useTransform(pointOnAxis);

return (
  <g className="x-axis-ticks">
    <line
      x1={xPx}
      y1={yPx - tickSize / 2}
      x2={xPx}
      y2={yPx + tickSize / 2}
      style={tickStyle}
    />
  </g>
);
```